PINBALL_GAME_BUTTONS_AND_LIGHTS.s

```diff
@@ -2960,14 +2960,14 @@ ENDRELDS	EQUALS
 # (SIMILARLY FOR PLEASE MARK).  FIRST PLACE THE CODED NUMBER FOR WHAT
 # ACTION IS DESIRED OF OPERATOR INTO THEREGISTERS REFERRED TO BY THE
 # `CHECKLIST' NOUN.  GO TO NVSUB WITH A DISPLAY VERB AND THE `CHECKLIST'
-# NOUN.  GO TO NVSUB AGAIN WTIH THE `PLEASE PERFORM' VERB AND ZEROS IN THE
+# NOUN.  GO TO NVSUB AGAIN WITH THE `PLEASE PERFORM' VERB AND ZEROS IN THE
 # LOW 7 BITS.  THIS `PASTES UP' THE `PLEASE PERFORM' VERB INTO THE VERB
 # LIGHTS.
 #
 # NVMONOPT IS AN ENTRY SIMILAR TO NVSUB, BUT REQUIRING AN ADDITIONAL
 # Page 370
 # PARAMETER IN L.  IT SHOULD BE USED ONLY WITH A MONITOR VERB-NOUN CODE IN
-# A.  AFTER EACH MONITOR DISPLAY A *PLEASE* VERB WILL BE PASED INT THE VERB
+# A.  AFTER EACH MONITOR DISPLAY A *PLEASE* VERB WILL BE PASSED INT THE VERB
```

Comment:
Neither are correct.

`PASTED` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0370.jpg)